docs/ops/deployment/overview.md

```diff
@@ -0,0 +1,358 @@
+*Application Mode* creates a session cluster per application and executes the application's `main()` 
+method on the cluster.
+
+
+
+## Vendor Solutions
+
+A number of vendors offer managed or fully hosted Flink solutions.
+None of these vendors are officially supported or endorsed by the Apache Flink PMC.
+Please refer to vendor maintained documentation on how to use these products. 
+
+<!--
+Please keep this list in alphabetical order
+-->
+
+#### AliCloud Realtime Compute
+
+[Website](https://www.alibabacloud.com/products/realtime-compute)
+
+Supported Environments:
+<span class="label label-primary">AliCloud</span>
+
+#### Amazon EMR
+
+[Website](https://aws.amazon.com/emr/)
+
+Supported Environments:
+<span class="label label-primary">AWS</span>
+
+#### Amazon Kinesis Data Analytics for Apache Flink
+
+[Website](https://docs.aws.amazon.com/kinesisanalytics/latest/java/what-is.html)
+
+Supported Environments:
+<span class="label label-primary">AWS</span>
+
+#### Cloudera DataFlow
+
+[Website](https://www.cloudera.com/products/cdf.html)
+
+Supported Environment:
+<span class="label label-primary">AWS</span>
+<span class="label label-primary">Azure</span>
+<span class="label label-primary">Google Cloud</span>
+<span class="label label-primary">On-Premise</span>
+
+#### Eventador
+
+[Website](https://eventador.io)
+
+Supported Environment:
+<span class="label label-primary">AWS</span>
+
+#### Huawei Cloud Stream Service
+
+[Website](https://www.huaweicloud.com/en-us/product/cs.html)
+
+Supported Environment:
+<span class="label label-primary">Huawei Cloud</span>
+
+#### Ververica Platform
+
+[Website](https://www.ververica.com/platform-overview)
+
+Supported Environments:
+<span class="label label-primary">AliCloud</span>
+<span class="label label-primary">AWS</span>
+<span class="label label-primary">Azure</span>
+<span class="label label-primary">Google Cloud</span>
+<span class="label label-primary">On-Premise</span>
+
+## Deployment Best Practices
+
+### How to provide dependencies in the classpath
+
+Flink provides several approaches for providing dependencies (such as `*.jar` files or static data) to Flink or user-provided
+applications. These approaches differ based on the deployment mode and target, but also have commonalities, which are described here.
+
+To provide a dependency, there are the following options:
+- files in the **`lib/` folder** are added to the classpath used to start Flink. It is suitable for libraries such as Hadoop or file systems not available as plugins. Beware that classes added here can potentially interfere with Flink, for example if you are adding a different version of a library already provided by Flink.
+
+- **`plugins/<name>/`** are loaded at runtime by Flink through separate classloaders to avoid conflicts with classes loaded and used by Flink. Only jar files which are prepared as [plugins]({% link ops/plugins.md %}) can be added here.
```

Comment:
```suggestion
- **`plugins/<name>/`** are loaded at runtime by Flink through separate classloaders to avoid conflicts with classes loaded and used by Flink. Only jar files which are prepared as [plugins]({% link deployment/filesystems/plugins.md %}) can be added here.
```